src/openforms/submissions/models.py

```diff
@@ -560,3 +560,6 @@ def delete(self, using=None, keep_parents=False):
 
     def get_display_name(self):
         return self.file_name or self.original_name
+
+    def get_format(self):
+        return os.path.splitext(self.get_display_name())[1].lstrip(".")
```

Comment:
Shouldn't this be a mime type? e.g. `application/pdf`

Reply:
All examples are extensions but the spec indeed prefers mime type so I modified the PR.